public/assets/sass/admin/generic-editor.scss

```diff
@@ -96,3 +96,39 @@
     }
   }
 }
+
+.modal {
+  display: none;
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+
+  .modal-content {
+    background-color: #fefefe;
+    margin: 15% auto; /* 15% from the top and centered */
+    padding: 20px;
+    border: 1px solid #888;
+    width: 80%; /* Could be more or less, depending on screen size */
+  }
+
+  .save-rich-text {
+    width: 100%;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+
+    > * {
+      margin: 10px;
+    }
+
+    .save-rich-text-btn {
+      justify-content: center;
+    }
+  }
+}
```

Comment:
Would it be possible for us to still have a visual indication that a node in the editor has been clicked, even if it is disabled?

Feel free to push this into its own ticket if its nontrivial but I think we should definitely provide the admin with some indication of what node they have selected.

Reply:
nontrivial, will create its own ticket after this gets merged